yeonjae/programmers-1/소수만들기.js

```diff
@@ -0,0 +1,26 @@
+function solution(nums) {
+  const numslength = nums.length;
+  let answer = 0;
+
+  for (let i = 0; i < numslength; i++) {
+    for (let j = i + 1; j < numslength; j++) {
+      for (let k = j + 1; k < numslength; k++) {
+        let sum = nums[i] + nums[j] + nums[k];
+
+        if (isPrime(sum)) {
+          answer++;
+        }
+      }
+    }
+  }
+
+  function isPrime(sum) {
+    if (sum < 2) return true;
+    for (let i = 2; i < sum; i++) {
+      if (sum % i == 0) return false;
+    }
+    return true;
+  }
+
+  return answer;
+}
```

Comment:
```suggestion
function solution(nums) {
  const numsLength = nums.length;
  let answer = 0;

  for (let i = 0; i < numslength; i++) {
    for (let j = i + 1; j < numsLength; j++) {
      for (let k = j + 1; k < numsLength; k++) {
        const sum = nums[i] + nums[j] + nums[k];

        if (isPrime(sum)) answer++;
      }
    }
  }

  function isPrime(sum) {
    for (let i = 2; i <= Math.sqrt(sum); i++) {
      if (sum % i == 0) return false;
    }
    return true;
  }

  return answer;
}
```
- 풀이 잘봤습니다 🙇‍♂️ 피드백을 드리자면 isPrime에서 `if (sum < 2)` 조건문은 해당 풀이에서 불 필요한 코드라 제거하였습니다. 또한, 반복 횟수가 sum까지 하는 것보다 `Math.sqrt(sum)`까지만 진행해도 문제가 없어서 반복 횟수를 최적화해주었습니다.
- numslength 변수 컨벤션이 잘못되었습니다 !! 카멜케이스로 `numsLength`로 수정해주었습니다
- 3중 for문에서 sum은 추가적인 로직이 없기 때문에 let말고 `const`로 선언해주시길 바랍니다 🙆‍♂️